src/pages/blog.tsx

```diff
@@ -37,6 +41,23 @@ const BlogPage = (): JSX.Element => (
       >
         {SECTION.blog}
       </Heading>
+      {/*TODO manage responsiveness (3 media instead of just 2)*/}
+      <Heading
+        as="h2"
+        color="primary"
+        fontSize={[2, 4]}
+        mb={[3, 5]}
+        textAlign="center"
+        style={centerHorizontally}
+      >
+        <Text width={[300, 700]} key="presentation">
+          {/*TODO move to content */}
+          Here you will find all the blog posts written about the Process
+          Analytics project. They are about how to use the libraries and tools
+          developed by the project, how we use some tools or external technical
+          components, ...
```

Comment:
I found there is too many "about" 😆 

A suggestion: 
```suggestion
Here you will find all the blog posts explaining how to use the libraries developed by Process Analytics, as well as those explaining how they were developed using external technical tools, and more.
```